viam-cartographer_test.go

```diff
@@ -0,0 +1,338 @@
+// Package viamcartographer_test tests the functions that require injected components (such as robot and camera)
```

Comment:
Why are we adding these tests when the ticket is about removing pre map representation endpoints?